react/testing_2/src/screens/Dashboard/components/UserList/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import UserRow from '../UserRow';
+
+import './styles.css';
+
+class UserList extends Component {
+  componentDidUpdate (){
+    console.log('UserList updated!');
+  }
+  render (){
+    return (
+      <div className={`user-list`}>
+        {this.props.users && this.props.users.map(user => <UserRow user={user} key={user.id} />)}
```

Comment:
I'd extract this arrow function. Surely they take this exercise as an example and compare what they did. I say it from experience haha

Reply:
idk, I don't think this is wrong